script/integration-tests

```diff
@@ -75,6 +75,9 @@ basic)
 kafka)
     bats tests/integration-tests-kafka.bats
     ;;
+nats)
+    bats tests/integration-tests-nats.bats
```

Comment:
add this tests as well in the `*` case (for GKE)

Reply:
I will do it in a seperate PR. Intergration tests for NATS are running as part of Minikube which should be sufficient for now.